tests/hdl/test_dsl.py

```diff
@@ -6,11 +6,11 @@
 from enum          import Enum
 from sys           import version_info
 
-from torii.hdl.ast import *
-from torii.hdl.cd  import *
-from torii.hdl.dsl import *
+from torii.hdl.ast import Cat, ClockSignal, Past, ResetSignal, Signal, SignalSet, Switch
+from torii.hdl.cd  import ClockDomain
+from torii.hdl.dsl import Module, SyntaxError, SyntaxWarning
 
-from ..utils       import ToriiTestSuiteCase
+from ..utils import ToriiTestSuiteCase
```

Comment:
This seems to have dropped the import alignment you set as the style for this code base.